src/wallet/wallet.cpp

```diff
@@ -4314,3 +4295,11 @@ CTxDestination CWallet::AddAndGetDestinationForScript(const CScript& script, Out
     default: assert(false);
     }
 }
+
+COutput CWallet::MakeOutput(const CWalletTx& wtx, int index, int depth, bool is_spendable, bool is_solvable, bool is_safe) const
+{
+    // If known and signable by the given wallet, compute input bytes
+    // Failure will keep this value -1
+    int input_bytes = is_spendable ? GetSpendSize(wtx, index) : -1;
```

Comment:
How about dropping `GetSpendSize` in favor of `CalculateMaximumSignedInputSize`? This is the only call. Previously `GetSpendSize` had fewer args but now they're basically equivalent.

Reply:
Or dropping `CalculateMaximumSignedInputSize` for some version of `GetSpendSize`, given the former is only called in `CWallet` contexts after this change.